packages/react-dom/src/events/SelectEventPlugin.js

```diff
@@ -72,32 +72,53 @@ function getSelection(node) {
       start: node.selectionStart,
       end: node.selectionEnd,
     };
-  } else if (window.getSelection) {
-    const selection = window.getSelection();
-    return {
-      anchorNode: selection.anchorNode,
-      anchorOffset: selection.anchorOffset,
-      focusNode: selection.focusNode,
-      focusOffset: selection.focusOffset,
-    };
+  } else {
+    const win =
+      (node.ownerDocument && node.ownerDocument.defaultView) || window;
+    if (win.getSelection) {
```

Comment:
Likewise, when is `getSelection` absent?

Reply:
Likewise, in IE8